genesyscloud/guide/resource_genesyscloud_guide_utils.go

```diff
@@ -0,0 +1,48 @@
+package guide
+
+import "net/http"
+
+func setRequestHeader(r *http.Request, p *guideProxy) *http.Request {
+	r.Header.Set("Content-Type", "application/json")
+	r.Header.Set("Accept", "application/json")
+	r.Header.Set("Authorization", "Bearer "+p.clientConfig.AccessToken)
+	return r
+}
+
+type DeleteObject struct {
```

Comment:
I think `DeleteObjectJob` is a better name. Makes it easier to understand what `Id` is referring to